src/components/common/constants.js

```diff
@@ -12,4 +12,11 @@ const pageDefault = {
   landing: "/",
 };
 
-export { dateDefault, pageDefault };
+const mapDefault = {
```

Comment:
I saw you are using a same name constant in `Map.tsx`, not this one. So is it redundant?

Reply:
Yes you are right, fixed